docs/ci/popperyml.md

```diff
@@ -67,15 +93,44 @@ popper init mypipe --env=alpine-3.4
 The above specifies that the pipeline named `mypipe` will be executed 
 inside a docker container using the `alpine-3.4` popper check image.
 
+## Metadata
+
+The `metadata` YAML entry specifies the set of data that gives information
+about the user's project. 
+
+It can be added using the `popper metadata --add command`
+
+For example :
+```bash
+popper metadata --add authors='Dennis Ritchie'
+```
+
+The above line adds a metadata entry 'authors' to the the project
+metadata.
+
+## Pipelines
+
+The `pipelines` YAML entry specifies the details for all the available
+pipelines. It gives us the information about -
+    *  the environment(s) in which a pipeline can be executed
+    *  the path to that pipeline
+    *  the various stages that are present in it 
+
+It gets updated automatically when a pipeline is initialized.
+
+## Popperized
+
+The `popperized` YAML entry specifies the list of available popperized 
+pipelines in a GitHub organization. By default, it points to the 
+`github/popperized` organization. The list forms the `search` dataset 
+for the `popper search` command.
+
 ## Stages
 
 The `stages` YAML entry specifies the sequence of stages that are 
 executed by the `popper check` command. By default, the `popper init` 
 command generates scaffold scripts for `setup.sh`, `run.sh`, 
-`post-run.sh`, `validate.sh`, `teardown.sh`. If any of those are not 
-present when the pipeline is executed using `popper check`, they are 
-just skipped (without throwing an error). At least one stage needs to 
-be executed, otherwise `popper check` throws an error.
+`post-run.sh`, `validate.sh`, `teardown.sh`.
```

Comment:
This is still the case:

> If any of those are not present when the pipeline is executed using `popper check`, they are just skipped (without throwing an error).

but should read `popper run` instead of `popper check`.

Reply:
@ivotron Changes made

Reply:
thanks. Sorry for the confusion. Besides changing `check` for `run`, I also meant that this phrase:

> If any of those are not present when the pipeline is executed using `popper check`, they are just skipped (without throwing an error).

still applies and should be part of the document.